lib/pymedphys/_experimental/pinnacle/rtstruct.py

```diff
@@ -296,7 +299,18 @@ def read_roi(ds, plan):
                     points[1] = round(points[1], 5)
                     points[2] = round(points[2], 5)
 
+                curr_points[0] = curr_points[0] \
+                    - (1*plan.ct_center[0]) \
+                    - (0*plan.iso_center[0])
```

Comment:
last part of these equations is multiplying zero * iso_center... Meaning this would always be zero and could be removed for readability?